homeassistant/components/roku/__init__.py

```diff
@@ -24,6 +27,10 @@
 ]
 _LOGGER = logging.getLogger(__name__)
 
+_T = TypeVar("_T", bound="SonarrSensor")
```

Comment:
@MartinHjelmare obv sonarr was copied here but with sharing between multiple types of platforms should it use RokuEntity?